frontends/ol-components/src/components/FormHelpers/FormHelpers.tsx

```diff
@@ -35,7 +35,7 @@ const Container = styled.div<{ fullWidth?: boolean }>(({ fullWidth }) => [
     display: "inline-flex",
     flexDirection: "column",
     alignItems: "start",
-    "> *": {
+    "> *:not(:last-child)": {
       marginBottom: "4px",
```

Comment:
This rule is supposed to make the 4px gap shown below. Note that there is no gap below the last child.

Figma:

<img width="864" alt="Screenshot 2024-08-29 at 1 28 06 PM" src="https://github.com/user-attachments/assets/53b14439-f87f-4302-a5d0-dedb86e838ee">